pkg/ddevapp/pantheon_test.go

```diff
@@ -0,0 +1,130 @@
+package ddevapp_test
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	. "github.com/drud/ddev/pkg/ddevapp"
+	"github.com/drud/ddev/pkg/testcommon"
+	"github.com/drud/ddev/pkg/util"
+	"github.com/stretchr/testify/assert"
+)
+
+/**
+ * These tests rely on an external test account managed by DRUD. To run them, you'll
+ * need to set an environment variable called "PANTHEON_API_KEY" with credentials for
+ * this account. If no such environment variable is present, these tests will be skipped.
+ */
+
+var pantheonTestSiteName = "ddev-test-site-do-not-delete"
+var pantheonTestEnvName = "bbowman"
+
+// TestConfigCommand tests the interactive config options.
+func TestPantheonConfigCommand(t *testing.T) {
+	if os.Getenv("PANTHEON_API_KEY") == "" {
+		t.Skip("No PANTHEON_API_KEY env var has been set. Skipping Pantheon specific test.")
+	}
+
+	// Set up tests and give ourselves a working directory.
+	assert := assert.New(t)
+	testDir := testcommon.CreateTmpDir("TestPantheonConfigCommand")
+
+	// testcommon.Chdir()() and CleanupDir() checks their own errors (and exit)
+	defer testcommon.Chdir(testDir)()
+	defer testcommon.CleanupDir(testDir)
```

Comment:
The order of the defers has to be swapped here, will fail on windows. Since defers are LIFO, this tries to remove the dir before chdir-ing out of it. Of course by now /me completely hates this pattern :)